mcs_benchmark_data/models/benchmark_antecedent.py

```diff
@@ -6,16 +6,24 @@
 from mcs_benchmark_data.models.benchmark_prompt import BenchmarkPrompt
 from mcs_benchmark_data.models.benchmark_context import BenchmarkContext
 
-from ..namespace import MCS
+from mcs_benchmark_data.namespace import MCS, SCHEMA
+from rdflib import Graph
 from rdflib.resource import Resource
 
 @dataclass_json(letter_case=LetterCase.CAMEL)
 @dataclass(frozen = True)
 class BenchmarkAntecedent(_Model):
     '''A list of elements that compose a benchmark sample'''
-    elements: Tuple[BenchmarkContext, BenchmarkPrompt]
+    elements: Union[Tuple[BenchmarkContext, BenchmarkPrompt], Tuple[BenchmarkPrompt,...]]
 
-    def to_rdf( self, *, add_to_resource: Resource) -> None:
-        if self.elements[0] is not None:
-            add_to_resource.add(MCS.BenchmarkContext, self.elements[0].text)
-        self.elements[1].to_rdf(add_to_resource = add_to_resource)
+    def to_rdf( self, *, graph: Graph) -> Resource:
+        resource = _Model.to_rdf(

```

Comment:
Your formatting is somewhat inconsistent. Have you set up Black in your editor?
https://black.readthedocs.io/en/stable/editor_integration.html